binary/cardano-binary.cabal

```diff
@@ -60,7 +60,12 @@ library
                        -Wredundant-constraints
                        -Wpartial-fields
                        -Wcompat
-
+  if impl(ghc >=8.8)
+    ghc-options:       -fno-warn-missing-deriving-strategies
+  if impl(ghc >=8.10)
+    ghc-options:       -fno-warn-missing-safe-haskell-mode
+                       -fno-warn-prepositive-qualified-module
+                       -fno-warn-unused-packages
```

Comment:
I prefer `-Wno-X` over `-fno-warn-X`, then it's consistent with the warning flags we use above.

Reply:
I have no opinion on this; I'm just taking the same approach as was used in `cardano-prelude`https://github.com/input-output-hk/cardano-prelude/blob/master/cardano-prelude.cabal#L79

Reply:
Yeah, I saw that too. Would you mind if I pushed a commit to this PR that changed it to `-Wno-X`?

Reply:
Not at all!

Reply:
Done